internal/controller/object/object.go

```diff
@@ -262,27 +298,34 @@ func (c *external) Create(ctx context.Context, mg resource.Managed) (managed.Ext
 		return managed.ExternalCreation{}, err
 	}
 
+	if e.applyExtractor != nil {
```

Comment:
we should still use `client.Create()`, otherwise, there is a danger that we actually patch an existing object.

Reply:
I think it is ok here different than the other context where we were discussing a similar problem with claims -> composites. 

Here, an Object is expected to take owner ship of the resource in the manifest. If it doesn't exist, `Create` will not be called at all. It it exist, `Update` will be called and we will patch to bring it to the desired state.